modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteClusterSnapshotHandlerTest.java

```diff
@@ -405,4 +407,34 @@ public void testHandlerSnapshotLocation() throws Exception {
             U.delete(snpDir);
         }
     }
+
+    /**
+     * Test ensures that snapshot fail if during check some files are absent.
```

Comment:
Minor: I would write like 'Test ensures that snapshot failS if some files are absent during the check.'

Reply:
Fixed.